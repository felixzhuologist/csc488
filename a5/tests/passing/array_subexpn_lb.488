{
    var numArray[-1 .. 5]: Integer
    var another: Integer
    numArray[1] := 11
    another := numArray[1]
    
    write "The value is "
    write another
}