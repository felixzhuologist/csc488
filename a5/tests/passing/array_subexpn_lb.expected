The value is 11